### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by rakeshrengaraj**
**Week-2:-Assignment-1: Responsive Webpage created with HTML and CSS**

Screenshot from 2022-08-26 15-32-24
Screenshot from 2022-08-26 15-34-00

#####################################################################################################

**Week-2:-Assignment-2: Animation using CSS**

Screenshot from 2022-08-26 15-39-36
Screenshot from 2022-08-26 15-39-32

### CODE REVIEW

File: Week-2/Assignment-1/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html />
+<html>
+
+<head>
```
Comment (sairamin) [inline Week-2/Assignment-1/index.html:4]: Indentation is missing.

File: Week-2/Assignment-1/index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    <main>
+        <figure>
+            <img src="https://www.w3schools.com/html/pic_trulli.jpg" alt="Trulli">
+            <figcaption>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore et dolore magna aliqua."</figcaption>
+        </figure>
+
+
+        <article>
```
Comment (sairamin) [inline Week-2/Assignment-1/index.html:37]: What is the difference between an article and a section ?

File: Week-2/Assignment-1/index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                </header>
+                <p>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text.</p>
+            </article>
+        </article>
+    </main>
+    <section>
+        <form>
+            <div class="form_input">
+                <label for="name">Full name:</label>
+                <input type="text" id="name" name="name" placeholder="Full name" required>
```
Comment (sairamin) [inline Week-2/Assignment-1/index.html:63]: **Suggestion:** Try to order the attributes in alphabetical order.

File: Week-2/Assignment-1/index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                <input type="Submit" value="Sign up">
+            </div>
+        </form>
+    </section>
+    <footer>
+        <p>© 2022 Lorem Ipsum</p>
+    </footer>
+</body>
+
+</html>
```
Comment (sairamin) [inline Week-2/Assignment-1/index.html:96]: Add a new line

File: Week-2/Assignment-1/style.css [hunk 5]
```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+html,
+body {
+    padding: 5px;
```
Comment (sairamin) [inline Week-2/Assignment-1/style.css:5]: Just use multiples of 4px

File: Week-2/Assignment-2/index.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    <div id="main">
+        <h1>Hi!!</h1>
+        <div id="top_left_div" class="child_div"></div>
+        <div id="top_right_div" class="child_div"></div>
+        <div id="bottom_left_div" class="child_div"></div>
+        <div id="bottom_right_div" class="child_div"></div>
+    </div>
+</body>
+
+</html>
```
Comment (sairamin) [inline Week-2/Assignment-2/index.html:21]: Add a new line

File: Week-2/Assignment-2/style.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+
+#main:hover #bottom_left_div {
+    transform: rotate(-90deg);
+}
+
+
+#main:hover #bottom_right_div {
+    transform: rotate(90deg);
+}
```
Comment (sairamin) [inline Week-2/Assignment-2/style.css:69]: Add a new line.

**Review by sairamin — CHANGES_REQUESTED** [review]
Stick to the same rules as mentioned in the Week 1 PR.

File: Week-2/Assignment-1/index.html (hunk 1 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-1/index.html:4]: Added Indentation with prettier code formatter

File: Week-2/Assignment-1/index.html (hunk 2 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-1/index.html:37]: The section tag is used to split the page into different sections like Introduction, Information, etc. Whereas the article tag contains independent content that doesn’t require any other context.

File: Week-2/Assignment-1/index.html (hunk 3 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-1/index.html:63]: Reordered attributes in alphabetical order

File: Week-2/Assignment-1/index.html (hunk 4 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-1/index.html:96]: Done

File: Week-2/Assignment-1/style.css (hunk 5 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-1/style.css:5]: Used multiples of 4px in padding, margin,width, height etc

File: Week-2/Assignment-2/index.html (hunk 6 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-2/index.html:21]: Done

File: Week-2/Assignment-2/style.css (hunk 7 above)
Comment (rakeshrengaraj) [reply Week-2/Assignment-2/style.css:69]: Done

**Review by sairamin — APPROVED** [review]

**Pull request merged**